debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/FiltersTest.java

```diff
@@ -162,41 +161,6 @@ public void shouldExcludeCollectionCoveredByLiteralInBlacklist() {
         assertCollectionIncluded("db2.collectionA");
     }
 
-    @Test(expected = ConfigException.class)
```

Comment:
FWIW, I think this approach using separate test methods was advantageous: if one fails, you know exactly which one it is; whereas now, you'll have to refer to the line number and see which one a failed assertion is about. Also, if there are multiple failures, all the affected test methods would have failed before, whereas now with the larger test methods, one failure may "hide" another failure, i.e. the latter will only become visible once the former has been fixed.

Reply:
You could provide custom failure messages too to recognize the failing test.

Reply:
This won't prevent the "hiding" issue, though.

Reply:
Also, you cannot re-run individual tests/assertions e.g. in your IDE. In general, finer-grained test methods are preferable for all these reasons.